recipes/mongo-cxx-driver/all/Findlibbson-1.0.cmake

```diff
@@ -0,0 +1,6 @@
+find_path(BSON_INCLUDE_DIR NAMES bson.h PATHS ${CONAN_INCLUDE_DIRS_MONGO-C-DRIVER})
```

Comment:
It would be nice to add these `FindXXX.cmake` files as a patch (if they are really needed).

Reply:
I deleted these files